rpc/serde/jsonrpc.go

```diff
@@ -92,6 +94,11 @@ type (
 	GetPaymentChannelsByLedgerResponse = []query.PaymentChannelInfo
 )
 
+type ReceiveVoucherResponse struct {
```

Comment:
We could just return the fromVoucher value and avoid having to create a new response struct and modifying the schema in nitro-gui.  If someone does want to know the total value they can just call`GetPaymentChannel`

Reply:
On the fence with this one -- I'm leaning towards making the breaking change, though. We shouldn't let the tail wag the dog.